Makefile

```diff
@@ -44,6 +44,11 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	pip-compile --upgrade -o requirements/quality.txt requirements/quality.in
 	pip-compile --upgrade -o requirements/test.txt requirements/base.in requirements/test.in
 	pip-compile --upgrade -o requirements/travis.txt requirements/travis.in
+	# Workaround for packages that break pip-tools due to python version specifiers somewhere upstream
+	# See https://github.com/nvie/pip-tools/issues/206
+	cat requirements/dev-extra.txt >> requirements/dev.txt
```

Comment:
I'm not sure.  Jeremy is aware of the pip-tools issue, and I think it's fairly high on his platter, but I'm not sure if it exists in jira or not.